src/api/spec/db/data/remove_duplicate_repositories_spec.rb

```diff
@@ -1,6 +1,5 @@
 require 'rails_helper'
 require Rails.root.join('db/data/20170306084550_remove_duplicate_repositories.rb')
-require Rails.root.join('db/migrate/20170306084558_change_repositories_remote_project_name_to_not_null.rb')
```

Comment:
I think this is not the only thing which can be changed/removed in this file. See the comment on the lines 5 to 7 in this file. There's also some funky stuff happening in the _before_ and _after_ blocks below.

Reply:
yes, well it's for spec and the spec is passing, so i guess, we are good?  The question is if we really need those 4 years old data migrations?

Reply:
It's good if the specs are already passing. If we keep the file around, I would still prefer to remove some of that weird code. It's going to confuse anybody looking into this.

It's true that we might not need those 4 years old data migrations anymore.

Reply:
Yes, I would like to dig in the left over in another PR, otherwise the changelog will be even bigger. I think this task can be done in the "follow-up PR" as I described in the PR description.